### PR TITLE
CSHARP-2976: Document How to do collations with LINQ.

### PR DESCRIPTION
**Pull request by DmitryLukyanov**

### CODE REVIEW

File: Docs/reference/content/reference/driver/crud/linq.md [hunk 1]
```diff
@@ -69,6 +69,15 @@ var query = collection.AsQueryable()
 ]
 ```
 
+Also, there is a list of aggregate options that can be applied for the whole pipeline via [`AggregationOptions`]({{< apiref "T_MongoDB_Driver_AggregateOptions" >}}). Below is an example of [`Collation`]({{< docsref "reference/collation/" >}}) configuring:
```
Comment (rstam) [inline Docs/reference/content/reference/driver/crud/linq.md]: Maybe fix grammar slightly:

    Below is an example of configuring a [`Collation`]({{< docsref "reference/collation/" >}}):

**Review by rstam — CHANGES_REQUESTED** [review]
Minor grammar change requested.

File: Docs/reference/content/reference/driver/crud/linq.md (hunk 1 above)
Comment (DmitryLukyanov) [reply Docs/reference/content/reference/driver/crud/linq.md]: done

**Comment by DmitryLukyanov** [comment]
ready for review

**Review by rstam — APPROVED** [review]
LGTM

File: Docs/reference/content/reference/driver/crud/linq.md [hunk 2]
```diff
@@ -69,6 +69,15 @@ var query = collection.AsQueryable()
...
 ```
 
+Also, there is a list of aggregate options that can be applied for the whole pipeline via [`AggregationOptions`]({{< apiref "T_MongoDB_Driver_AggregateOptions" >}}). Below is an example of configuring a [`Collation`]({{< docsref "reference/collation/" >}}):
+
+```csharp
+var collation = new Collation("en_US", strength: CollationStrength.Secondary);
+var aggregateOptions = new AggregateOptions { Collation = collation };
+var query = collection
+    .AsQueryable(aggregateOptions)
+    .Where(x => p.Name == "tom");
```
Comment (MikalaiMazurenka) [inline Docs/reference/content/reference/driver/crud/linq.md]: `x` should be `p`

**Review by MikalaiMazurenka — CHANGES_REQUESTED** [review]
Minor changes requested.

File: Docs/reference/content/reference/driver/crud/linq.md (hunk 2 above)
Comment (DmitryLukyanov) [reply Docs/reference/content/reference/driver/crud/linq.md]: good catch, done

**Review by MikalaiMazurenka — APPROVED** [review]
LGTM

**Pull request closed without merging**